lib/feature/add_seed_to_app/create_seed/view/create_seed_view.dart

```diff
@@ -0,0 +1,208 @@
+import 'package:app/feature/add_seed_to_app/create_seed/create_seed.dart';
+import 'package:app/generated/assets.gen.dart';
+import 'package:app/l10n/l10n.dart';
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:ui_components_lib/ui_components_lib.dart';
+
+/// Callback that calls if user taps skip checking or check phrase.
+typedef CreateSeedNavigationCallback = void Function(
+  BuildContext context,
+  List<String> phrase,
+);
+
+/// {@template create_seed_view}
+/// Widget that allows user to create random seed phrase, copy it and check
+/// {@endtemplate}
+class CreateSeedView extends StatefulWidget {
+  /// {@macro create_seed_view}
+  const CreateSeedView({
+    required this.skipCallback,
+    required this.checkCallback,
+    super.key,
+  });
+
+  final CreateSeedNavigationCallback skipCallback;
+  final CreateSeedNavigationCallback checkCallback;
+
+  @override
+  State<CreateSeedView> createState() => _CreateSeedViewState();
+}
+
+class _CreateSeedViewState extends State<CreateSeedView> {
+  late final cubit = CreateSeedCubit()..init();
+
+  final isCopied = ValueNotifier<bool>(false);
```

Comment:
* We already have cubit (why not bloc?)
* Aaaand we use ValueNotifier, that complicates the UI code. 

Maybe consider to move all business logic from UI into cubit/bloc?

Reply:
Cubit here is just a single (not repeatable action), bloc is too heavy for this.
ValueNotifier is just a single flag to reduce widget tree rebuilding. Don't think heavy cubit/bloc need to be here

Reply:
But how you will test the widget with any logic inside it? With external state you can render any state, and it will be consistent.

Reply:
-